src/convertToObject.js

```diff
@@ -6,7 +6,30 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0);
+
+  const styleObject = {};
+
+  for (let i = 0; i < array.length; i++) {
+    const style = array[i];
+
+    // Separar a chave (propriedade) e o valor da declaração
+    const [key, value] = style.split(':');
+
+    if (key && value) {
+      // Remover espaços extras da chave e do valor
+      const cleanedKey = key.trim();
+      const cleanedValue = value.trim();
+
+      // Adicionar o par chave-valor ao objeto
+      styleObject[cleanedKey] = cleanedValue;
+    }
+  }
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. You can use the `reduce` method to transform the array of style strings into an object.